From 74cdadd0b0f2208b99dba0e987fffa39064f9b0e Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Mon, 28 May 2018 16:27:08 +0800
Subject: [PATCH] ASoC: rk3308_codec: reduce delay for reinit-mic

The vendor suggest that we just need to use delay some
micro seconds for reinit-mic more quickly.

Change-Id: I05cd37828bcd594ca6ab634fb8b5505e01afe130
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 sound/soc/codecs/rk3308_codec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rk3308_codec.c b/sound/soc/codecs/rk3308_codec.c
index 9387cd646f9c..a6d2aa57863c 100644
--- a/sound/soc/codecs/rk3308_codec.c
+++ b/sound/soc/codecs/rk3308_codec.c
@@ -1797,7 +1797,7 @@ static int rk3308_codec_adc_reinit_mics(struct rk3308_codec_priv *rk3308)
 				   RK3308_ADC_CH2_MIC_INIT);
 	}
 
-	usleep_range(2000, 2500);	/* estimated value */
+	usleep_range(200, 250);		/* estimated value */
 
 	for (grp = 0; grp < ADC_LR_GROUP_MAX; grp++) {
 		/* vendor step 1 */
-- 
2.35.3

